ioc_copier/ioc_copier.py

```diff
@@ -0,0 +1,240 @@
+        text_lines[i] = temp_text
+        temp_text = re.sub(f"IOC-{start_copy}", f"IOC-{current_copy}", text_lines[i])
+        text_lines[i] = temp_text
+        temp_text = re.sub(f"{ioc}_{start_copy}", f"{ioc}_{current_copy}", text_lines[i])
+        text_lines[i] = temp_text
+        temp_text = re.sub(f"RAMPFILELIST{start_copy}", f"RAMPFILELIST{current_copy}", text_lines[i])
+        text_lines[i] = temp_text
+
+        temp_text = re.sub(f"IOC_0{start_copy}", f"IOC_{padded_current_copy}", text_lines[i])
+        text_lines[i] = temp_text
+        temp_text = re.sub(f"IOC-0{start_copy}", f"IOC-{padded_current_copy}", text_lines[i])
+        text_lines[i] = temp_text
+        temp_text = re.sub(f"{ioc}_0{start_copy}", f"{ioc}_{padded_current_copy}", text_lines[i])
+        text_lines[i] = temp_text
+        temp_text = re.sub(f"RAMPFILELIST0{start_copy}", f"RAMPFILELIST{padded_current_copy}",
+                           text_lines[i])
+        text_lines[i] = temp_text
+
+
+def help_check():
+    """
+    Function to handle printing help.
+    """
+    if "-h" in sys.argv:
+        print("First Argument: <ioc-to-duplicate>")
+        print("This should be the name of the ioc folder, the folders to "
+              "actual duplicate will all contain this in their names.\n")
+        print("Second Argument: <number-ioc-to-copy")
+        print("This should be the last currently existing ioc number, "
+              "e.g. 2. IOC-01 should not be copied in this way, as later"
+              " iocs should reference some of its files rather than copy them.\n")
+        print("Third Argument: <first-copy>")
+        print("This should be the number of the first copy, i.e."
+              " the first IOC made will be IOC-<first-copy>.\n")
+        print("Fourth Argument: <max-number-ioc>")
+        print("This should be the maximum number copied to.\n")
+        sys.exit()
+
+
+def handle_arguments():
+    """
+    Function to handle arguments of ioc_copier.py.
+    Returns:
+        ioc - The name of the ioc to run, folders to copy will be of the form,
+              <ioc>-IOC-01App and ioc<ioc>-IOC-01.
+        start_copy - The folder to copy (avoid IOC1 as other IOCs usually reference
+                     this rather than copying it directly).
+        initial_copy - The first copy.
+        max_copy - The final copy.
+    """
+    if len(sys.argv) < 5:
+        print("Not enough arguments")
+        print("Arguments should be <ioc-to-duplicate> <number-ioc-to-copy> "
+              "<first-copy> <max-number-ioc>")
+        print("use argument \"-h\" for more details.")
+        sys.exit()
+    elif len(sys.argv) > 5:
+        print("Too many arguments")
+        print("Arguments should be <ioc-to-duplicate> <number-ioc-to-copy> "
+              "<first-copy> <max-number-ioc>")
+        print("use argument \"-h\" for more details.")
+        sys.exit()
+    else:
+        ioc = sys.argv[1]
+        global start_copy
```

Comment:
Constants should conform to upper case naming style and because it's a global variable, should be defined at module level. This will fail pylint checks under `invalid-name` and also `global-variable-undefined`